code/game/area/areas.dm

```diff
@@ -497,8 +497,10 @@ var/list/teleportlocs = list()
 	if(!T || !isturf(T))
```

Comment:
@Cyberboss it doesnt' really matter tbh, the typepaths are just guidelines unless you actually check the istype